src/tv2-common/helpers/graphics/internal/InternalGraphic.ts

```diff
@@ -108,7 +108,8 @@ export abstract class InternalGraphic extends Graphic {
 				type: Tv2PieceType.GRAPHICS,
 				outputLayer: Tv2OutputLayer.OVERLAY,
 				partType: this.partDefinition?.type,
-				pieceExternalId: this.partDefinition?.externalId
+				pieceExternalId: this.partDefinition?.externalId,
+				graphicTemplateName: this.templateName
```

Comment:
To have it conform to the other uses of naming attributes for graphics (in sofie server), the attribute should be `graphicsTemplateName`.

Reply:
Changed to `graphicsTemplateName`.